topic_folders/lesson_development/cac-consult-rubric.md

```diff
@@ -1,7 +1,7 @@
 ## Curriculum Advisory Committee Consultation Rubric
 
 This rubric defines the division of responsibilities between The Carpentries Maintainers and The Carpentries Curriculum Advisory Committees (CACs). 
-[Roles and responsibilities](https://docs.carpentries.org/topic_folders/lesson_development/lesson_development_roles.html#curriculum-advisory-committees) 
+[Roles and responsibilities](https://docs.carpentries.org/topic_folders/lesson_development/curriculum_advisory_committees.html) 
```

Comment:
```suggestion
[Roles and responsibilities](https://docs.carpentries.org/topic_folders/lesson_development/curriculum_advisory_committees.html#roles-and-responsibilities) 
```

I think the link could be further improved by taking the visitor directly to the relevant section on roles and responsibilities of CACs